chartfx-chart/src/main/resources/de/gsi/chart/solid-pick.css

```diff
@@ -119,269 +119,202 @@ is being used to size a border should also be in pixels.
-        -fx-background,
-        black 10%,
-        derive(-fx-background, -15%) 30%
-    );
+  -fx-text-box-border: ladder(-fx-background, black 10%, derive(-fx-background, -15%) 30%);
 
-    /* Lighter than -fx-background and used to provide a small highlight when
+  /* Lighter than -fx-background and used to provide a small highlight when
      * needed on top of -fx-background. This is never a shadow in Modena but
      * keep -fx-shadow-highlight-color name to be compatible with Caspian.
      */
-    -fx-shadow-highlight-color: ladder(
-        -fx-background,
-        rgba(255,255,255,0.07) 0%,
-        rgba(255,255,255,0.07) 20%,
-        rgba(255,255,255,0.07) 70%,
-        rgba(255,255,255,0.7) 90%,
-        rgba(255,255,255,0.75) 100%
-      );
-
-    /* A gradient that goes from a little darker than -fx-color on the top to
+  -fx-shadow-highlight-color: ladder(-fx-background, rgba(255, 255, 255, 0.07) 0%, rgba(255, 255, 255, 0.07) 20%,
+    rgba(255, 255, 255, 0.07) 70%, rgba(255, 255, 255, 0.7) 90%, rgba(255, 255, 255, 0.75) 100%);
+
+  /* A gradient that goes from a little darker than -fx-color on the top to
      * even more darker than -fx-color on the bottom.  Typically is the second
      * color in the -fx-background-color list as the small thin border around
      * a control. It is typically the same size as the control (i.e., insets
      * are 0).
      */
-    -fx-outer-border: derive(-fx-color,-23%);
+  -fx-outer-border: derive(-fx-color, -23%);
 
-    /* A gradient that goes from a bit lighter than -fx-color on the top to
+  /* A gradient that goes from a bit lighter than -fx-color on the top to
      * a little darker at the bottom.  Typically is the third color in the
      * -fx-background-color list as a thin highlight inside the outer border.
      * Insets are typically 1.
      */
-    -fx-inner-border: linear-gradient(to bottom,
-                ladder(
-                    -fx-color,
-                    derive(-fx-color,30%) 0%,
-                    derive(-fx-color,20%) 40%,
-                    derive(-fx-color,25%) 60%,
-                    derive(-fx-color,55%) 80%,
-                    derive(-fx-color,55%) 90%,
-                    derive(-fx-color,75%) 100%
-                ),
-                ladder(
-                    -fx-color,
-                    derive(-fx-color,20%) 0%,
-                    derive(-fx-color,10%) 20%,
-                    derive(-fx-color,5%) 40%,
-                    derive(-fx-color,-2%) 60%,
-                    derive(-fx-color,-5%) 100%
-                ));
-    -fx-inner-border-horizontal: linear-gradient(to right, derive(-fx-color,55%), derive(-fx-color,-5%));
-    -fx-inner-border-bottomup: linear-gradient(to top, derive(-fx-color,55%), derive(-fx-color,-5%));
-
-    /* A gradient that goes from a little lighter than -fx-color at the top to
+  -fx-inner-border: linear-gradient(to bottom, ladder(-fx-color, derive(-fx-color, 30%) 0%, derive(-fx-color, 20%) 40%,
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line function (function-comma-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)